package.json

```diff
@@ -10,7 +10,9 @@
     "check-ts": "tsc --noEmit && tsc --project test/tsconfig.json --noEmit",
     "check-lint": "tsc --noEmit && eslint 'src/**/*.ts' 'test/**/*.ts'",
     "check-format": "prettier -c 'src/**/*.ts' 'test/**/*.ts'",
-    "check-spell": "cspell 'src/**/*ts' 'test/**/*ts'"
+    "check-spell": "cspell 'src/**/*ts' 'test/**/*ts'",
+    "types:prepare": "rm -rf dist && tsc --declaration true --sourceMap false && cp package.json dist/package.json && cp README.md dist/README.md && cd dist && npm version $(git describe --tags --always)",
+    "types:publish": "cd dist && npm publish --access public"
```

Comment:
I have always published the entire package not only dist. Are you sure it's a good idea publish only `dist`